### DISCUSSION TITLE
Avoid splitting gt() tables in pdf-rendering

### DISCUSSION THREAD

**Discussion by ca-signorelli** [quarto-dev/quarto-cli#2473] [Q&A] [answered]
I use Quarto and make tables using the gt() package. I render the output to pdf.
Is there a way to avoid that tables that could fit to one page is split between two pages?
Thank you.

**Comment by cscheid** [comment, marked as answer]
This is an issue with Pandoc, unfortunately. See https://groups.google.com/g/pandoc-discuss/c/znkTLPkekOg/m/KH6SplLXi7gJ

If you want to work around it, the suggestion on the thread is to use a Lua filter to replace `longtable` with `tabular`.

**Reply by ca-signorelli** [reply]
Thank you for following up on this question.

**Reply by nathancarter** [reply]
That google group is not open to all.  A web search didn't make it obvious to me how to write the kind of Lua filter you're talking about.  Can anyone point in the direction of a usable piece of Lua code?  Thank you.

**Reply by mcanouil** [reply]
@nathancarter Prefer opening new discussions with sufficient context rather than resurrecting very old threads that are very likely outdated.

**Reply by cderv** [reply]
@nathancarter Pandoc has moved from google group to discussion:
- https://github.com/jgm/pandoc/issues/9250

This is why the pandoc-discuss is block now. 

For your problem, do look into **gt** and latest quarto, share an example, things may have evolved and there could be solution now.
